scripts/install_singularity.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ex
+
```

Comment:
we can also check that if `RUNTIME` is not `singularity`, we don't need to do anything